code/game/turfs/openspace/openspace.dm

```diff
@@ -0,0 +1,62 @@
+/turf/open/openspace
+	name = "open space"
+	desc = "Watch your step!"
+	icon_state = "grey"
+	baseturfs = /turf/open/openspace
+	CanAtmosPassVertical = ATMOS_PASS_YES
+	plane = FLOOR_OPENSPACE_PLANE
+	layer = OPENSPACE_LAYER
+	//mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+
+/turf/open/openspace/debug/update_multiz()
+	..()
+	return TRUE
+
+/turf/open/openspace/Initialize()
+	. = ..()
+	return INITIALIZE_HINT_LATELOAD
+
+/turf/open/openspace/LateInitialize()
+	update_multiz(TRUE)
+
+/turf/open/openspace/Destroy()
+	vis_contents.Cut()
+	return ..()
+
+/turf/open/openspace/update_multiz(prune_on_fail = FALSE)
+	. = ..()
+	vis_contents.Cut()
+	var/turf/T = below()
+	if(!T)
+		if(prune_on_fail)
+			ChangeTurf(/turf/open/floor/plating)
+		return FALSE
+	vis_contents += T

```

Comment:
@Iamgoofball not sure how I'd darken just the vis contents instead of everything on the tile. I ended up doing a filter thing as everything else looked like crap.

Reply:
Dwarf fortress doesn't even render lower z-levels, the only time it renders z-levels other than the one you're on is in adventure mode, where you're given two little 5x5 windows in the top-right corner of your screen that show z-levels directly above and below you.
Shrinking vis_contents to add a parallax effect would simply be impossible due to the limitations of vis_contents, as you cannot control the layer or plane of things rendered via vis_contents.
Darkening vis_contents can be done by using the keep_together appearance flag on the turf and changing its color.